Dashboard/app/js/lib/controllers/survey-controllers.js

```diff
@@ -977,6 +960,69 @@ FLOW.questionControl = Ember.ArrayController.create({
     }
   }.observes('FLOW.selectedControl.selectedQuestion'),
 
+  reorderQuestions: function (qgId, reorderPoint, reorderDirection) {
```

Comment:
for the `reorderDirection` I suggest we use the commonly known terms i.e. `increment` and `decrement`.  Also is it `reorderDirection` or `reorderOperation`?